services/user-tenant-service/src/main/java/com/sourcefuse/jarc/services/usertenantservice/commons/TypeRole.java

```diff
@@ -0,0 +1,68 @@
+package com.sourcefuse.jarc.services.usertenantservice.commons;
+
+import static com.sourcefuse.jarc.services.usertenantservice.enums.RoleType.ADMIN;
+import static com.sourcefuse.jarc.services.usertenantservice.enums.RoleType.DEFAULT;
+import static com.sourcefuse.jarc.services.usertenantservice.enums.RoleType.SUPER_ADMIN;
+
+import com.sourcefuse.jarc.services.usertenantservice.enums.RoleType;
+import java.util.Collections;
+import java.util.EnumMap;
+import java.util.List;
+import java.util.Map;
+
+public final class TypeRole {
```

Comment:
What is this for ?

Reply:
This class contain static map to store the permission that i had used in TenantUserService BL ,
but will used it later Removed as of now